### PR TITLE
Added methods that return the Judo UIViewController instances

### PR DESCRIPTION
**Pull request by mpetrenco**
### Note:
Not too happy about presenting the Apple Pay sheet in the Interactor, but the alternative is dragging the JPApplePayService and completion handler all over the place.

### CODE REVIEW

**Comment by AlexeiJovmir** [comment]
Hmm. @mpetrenco  our CI failed with weird error.

File: Source/JudoKit.m [hunk 1]
```diff
@@ -152,6 +184,19 @@ - (void)invokePaymentMethodScreenWithMode:(JPTransactionMode)mode
                                                     completion:nil];
 }
 
+- (UIViewController *)paymentMethodViewControllerWithMode:(JPTransactionMode)mode
+                                            configuration:(JPConfiguration *)configuration
+                                               completion:(JPCompletionBlock)completion {
+    UIViewController *controller;
+    controller = [JPPaymentMethodsBuilderImpl buildModuleWithMode:mode
```
Comment (stpopa) [inline Source/JudoKit.m:191]: can we just:
```
    return [JPPaymentMethodsBuilderImpl buildModuleWithMode:mode
                                                    configuration:configuration
                                               transactionService:self.transactionService
                                            transitioningDelegate:self.transitioningDelegate
                                                completionHandler:completion];

```

File: Source/JudoKit.m [hunk 2]
```diff
@@ -90,12 +90,30 @@ - (void)invokeTransactionWithType:(JPTransactionType)type
...
+    if (!controller) {
+        return;
+    }
+    
+    [UIApplication.topMostViewController presentViewController:controller
+                                                      animated:YES
+                                                    completion:nil];
+}
+
+- (UIViewController *)transactionViewControllerWithType:(JPTransactionType)type
```
Comment (stpopa) [inline Source/JudoKit.m:106]: we're `return nil;` on line 116, please reflect this in method signature with a `nullable` annotation

File: Source/JudoKit.m [hunk 3]
```diff
@@ -104,46 +122,60 @@ - (void)invokeTransactionWithType:(JPTransactionType)type
...
+    if (!controller) {
+        return;
+    }
+    
+    [UIApplication.topMostViewController presentViewController:controller
+                                                      animated:YES
+                                                    completion:nil];
+}
+
+- (UIViewController *)applePayViewControllerWithMode:(JPTransactionMode)mode
```
Comment (stpopa) [inline Source/JudoKit.m:149]: we're return nil; on line 157, please reflect this in method signature with a nullable annotation

**Comment by mpetrenco** [comment]
@AlexeiJovmir the CI issue has been fixed in the Carthage PR. It's due to an outdated slather.yml

**Pull request merged**